scripts/package_release.sh

```diff
@@ -17,7 +17,14 @@ charttmpdir=$(mktemp -d 2>/dev/null || mktemp -d -t charttmpdir)
 charttmpdir=${charttmpdir}/chart
 
 cp -R deploy/chart/ "${charttmpdir}"
-sed -i "s/^[Vv]ersion:.*\$/version: ${version}/" "${charttmpdir}/Chart.yaml"
+
+# A space character is needed for -i argument in mac BSD sed.
+# GNU sed has a --version argument, and BSD sed does not. 
+# Check for both conditions to be safe for darwin users that override sed with gsed
+if [ -z "${OSTYPE##*"darwin"*}" ] && ! sed --version >/dev/null 2>&1; then
+    SP=" "
+fi
+sed -i"${SP}"'' "s/^[Vv]ersion:.*\$/version: ${version}/" "${charttmpdir}/Chart.yaml"
```

Comment:
Since we have a separate source and destination file, how about avoiding `-i` altogether with something like this?

```suggestion
sed "s/^[Vv]ersion:.*\$/version: ${version}/" deploy/chart/Chart.yaml > "${charttmpdir}/Chart.yaml"
```

Reply:
good point... this is easier and accomplishes the same goal. Fixed.